theano/sandbox/cuda/cula.py

```diff
@@ -0,0 +1,127 @@
+import theano
+from theano.sandbox.cuda.type import CudaNdarrayType
+from theano.sandbox.cuda import GpuOp
+
+from theano.sandbox.cuda.basic_ops import as_cuda_ndarray_variable
+from theano.sandbox.cuda import cuda_ndarray
+
+dimshuffle = cuda_ndarray.cuda_ndarray.dimshuffle
+
+try:
+    from scikits.cuda import cula
+    cula_available = True
+except (ImportError, OSError):
+    pass
```

Comment:
@caglar added the initialization of `cula_available` to `False` a couple of lines earlier, so it has actually been addressed alread.